src/components/content-box/_content-box.scss

```diff
@@ -34,7 +34,7 @@ $include-html: false !default;
 
     &--spaced {
       .mint-content-box__content, .mint-content-box__footer {
-        @media (min-width: $breakpointMedium) {
+        @include mintBreakpoint(medium-up) {
```

Comment:
so you think in here it would be better to have it in camel case and just in classes use it like hyphen ? (like small-only) ?